web/src/lib/components/shared-components/gallery-viewer/gallery-viewer.svelte

```diff
@@ -65,6 +66,20 @@
     }
   };
 
+  const handleAction = (action: AssetAction, asset: AssetResponseDto) => {
+    switch (action) {
+      case AssetAction.DELETE:
```

Comment:
Should we handle the archive action as well like this?

Reply:
Yes, archieve is added. Do you think we should also navigate to next photo for action arvhieve?

Reply:
Yes, archieve is added. Do you think we should also navigate to next photo for action arvhieve?

Reply:
The idea is that if you are on the viewer and click "archive", "delete" or "trash" the asset no longer belongs to the current collection, so auto navigating forward seems to make a lot of sense. So yeah, I would have all of them to that same action.

Reply:
It turns out that it is better if the gallery component handles the navigation to next/previous, in order to not double up on events (the asset grid also does navigation on trash/delete - the parent component of the asset viewer in some situations)